sql/src/main/java/io/crate/analyze/UnboundAnalyzer.java

```diff
@@ -271,5 +278,14 @@ public AnalyzedStatement visitOptimizeStatement(OptimizeStatement<?> node, Analy
                 context.transactionContext()
             );
         }
+
+        public AnalyzedDropTable visitDropTable(DropTable<?> node, Analysis context) {
+            return dropTableAnalyzer.analyze(node,context.sessionContext());
+        }
+
+        public AnalyzedStatement visitDropBlobTable(DropBlobTable<?> node, Analysis analysis) {
+            return visitNode(node, analysis);
```

Comment:
should be `dropTableAnalyzer.analyze(node, context.sessionContext());` I guess..